boostregex/UTF8DocumentIterator.h

```diff
@@ -12,80 +27,86 @@ namespace Scintilla {
 
 class Document;
 
-class UTF8DocumentIterator : public std::iterator<std::bidirectional_iterator_tag, wchar_t>
+class UTF8DocumentIterator
 {
 public:
-		UTF8DocumentIterator() {};
-
-		UTF8DocumentIterator(Document* doc, Sci::Position pos, Sci::Position end);
-		UTF8DocumentIterator(const UTF8DocumentIterator& copy);
-
-		bool operator == (const UTF8DocumentIterator& other) const
-		{
-				return (ended() == other.ended()) && (m_doc == other.m_doc) && (m_pos == other.m_pos);
-		}
-
-		bool operator != (const UTF8DocumentIterator& other) const
+	using iterator_category = std::bidirectional_iterator_tag;
+	using value_type = wchar_t;
+	using difference_type = ptrdiff_t;
+	using pointer = wchar_t*;
+	using reference = wchar_t&;
+
+	UTF8DocumentIterator() {};

```

Comment:
This is just a whitespace change.
I changed 2 tabs to 1 tab to make it consistent with other code..